### ISSUE TITLE
Question why the need for a "Turn Away" button for "Takes Calls Directly" conference

### ISSUE THREAD

**Issue by drodruck** [josephr5000/confdb-public#385]
The "Add Turn Away" button should be grayed out just like the "Add New Request" button is for Conferences that take calls directly:
![image](https://cloud.githubusercontent.com/assets/20655031/22910138/e08fd91a-f20c-11e6-9278-cb415dc31d9b.png)

**Comment by josephr5000** [comment]
This looks like the view when logged into the Call Center.  In general I leave Add Turn Away available even when the conference is not available for CC referrals (closed, quota full, or call directly) so the CC have the opportunity to record that someone called and they couldn't help directly.

I would agree this is most useful for the case where the conference is closed or the quota is full, as the CC has reason to believe the client won't be helped, whereas with "Calls Directly" the CC doesn't really know.  By the same token, now that the CC passes all calls to conferences, they have equally little visibility whether clients are being helped.

Given that availability of this button isn't entirely unreasonable, I'm curious what made you raise this issue now?  Did someone complain about it?  Or...?

**Comment by drodruck** [comment]
Yes, I got an email from the member currently on call who did not understand what this message ment:
![image](https://cloud.githubusercontent.com/assets/20655031/22914295/91cdf204-f224-11e6-82cb-82e7fb3e911d.png)

It creates confusion among the ranks since the client is being helped, not turned away.  It adds a note that is unnecessary and confusing (which, of course, we can delete).  

I don't see the need for it from the Call Center's point of view or ours.

**Comment by josephr5000** [comment]
Got it.  Does this happen often?  Occasionally?  Rarely?

That particular example is clearly operator error on the CC end.  (1) That client definitely wasn't turned away, they were just asked to call you directly, and (2) Even if you consider that a flavor of turning away, by no means is there No Active Conference in your area.

The broader question is to rethink turnaways given how radically the Council has reshaped CC operations since we designed this behavior.  That's really part of the 2017 update discussions that are beginning.

So...back to my question how frequently this happens, which will help in prioritizing this.

**Comment by drodruck** [comment]
I agree, this was most likely operator error.  

This is the first time it has happened so I don't know whether it will be occasional or rare, probably not often.  

My point is that the Call Center will never turn away one of our clients, just refer them on to us.  So I don't see the need for the button.

**Comment by josephr5000** [comment]
No disagreement there, Doug.  Like I said, it's a question of prioritizing, so my limited time is focused on things with the most bang for the buck.  

Unless/until this happens more often, I'm going to file this as an annoyance but not a high priority.

**Comment by drodruck** [comment]
Okay.....I will let you know if this becomes more than an annoyance.

**Comment by josephr5000** [comment]
Downgraded priority to blue-sky, but added to 2018 project list so it'll get done if we have time.
